Containerfile.cli-artifacts

```
@@ -8,31 +8,38 @@ COPY ./internal internal
 COPY ./go.* ./
 COPY ./pkg pkg
 COPY ./test test
+COPY ./packaging packaging
 COPY ./Makefile .
 # make sure that version extraction works
 COPY .git .git
 
 USER 0
-RUN make multiarch-build-cli
+RUN dnf install --nodocs -y jq
+RUN make build-multiarch-clis
 
 FROM registry.access.redhat.com/ubi9/ubi as certs
-RUN dnf install --nodocs -y nginx && dnf clean all
+RUN dnf install --nodocs -y nginx && \
+    rm -f /etc/nginx/nginx.conf /etc/nginx/conf.d/default.conf && \
+    dnf clean all
 
 
 ENV USER_UID=1001 \
     USER_NAME=server \
     HOME=/home/server \
-    NGINX_CONF_PATH=/etc/nginx/nginx.conf
+    NGINX_CONF_PATH=/etc/nginx/nginx.conf \
+    ENTRYPOINT_PATH=/usr/local/bin/entrypoint.sh
 
-COPY packaging/containers/cli-artifacts/nginx.conf ${NGINX_CONF_PATH}
+COPY --from=builder /app/packaging/containers/cli-artifacts/nginx.conf ${NGINX_CONF_PATH}
+COPY --from=builder /app/packaging/containers/cli-artifacts/entrypoint.sh ${ENTRYPOINT_PATH}
 
 USER root
 
 RUN mkdir -p ${HOME} && \
+    mkdir -p ${HOME}/src/gh-archives \
```

just one `mkdir -p` should be enough
```suggestion
RUN mkdir -p ${HOME}/src/gh-archives \
```